pegasus/sites.v3/hourofcode.com/views/home_signup.haml

```diff
@@ -4,9 +4,7 @@
 %a#hocsignupform{name: 'signup'}
 #join-us-header=hoc_s(:front_join_us_button)
 - if ["actual-hoc", "post-hoc", false].include?(hoc_mode)
-  #signup-closed
-    !=hoc_s(:signup_registration_not_required_markdown, markdown: :inline, locals: {hoc_activities_url: resolve_url('/learn'), howto_guide_url: resolve_url('/how-to')})
-    =hoc_s(:signup_registration_closed)
```

Comment:
It looks like there was already "registrations closed" messaging here before we did the homepage refresh (https://github.com/code-dot-org/code-dot-org/pull/48187) — I deprecated this to add the updated copy.

Reply:
Have we confirmed this the copy we now want through June? (It seems like this changes the copy that will displayed in post-hoc and HoC off-season too?)